composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cascade-energy/distributed-locking",
+  "description": "Base distributed locking library",
+  "minimum-stability": "dev",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Charles Berube",
+      "email": "charles.berube@cascadeenergy.com"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "Cascade\\": "src/"
```

Comment:
Could eliminate the extra directory by just mapping `src` to `"Cascade\\DistributedLocking\\"`. Any reason not to?